src/services/query.ts

```diff
@@ -0,0 +1,26 @@
+import { PLUGIN_ID } from './routing';
+
+export const buildLogVolumeQuery = (expr: string, overrides?: Record<string, unknown>) => {
+  return {
+    legendFormat: '{{level}}',
+    expr,
+    ...defaultQueryParams,
+    ...overrides,
+  };
+};
+
+export const buildLogQuery = (expr: string, overrides?: Record<string, unknown>) => {
+  return {
+    maxLines: 1000,
```

Comment:
AFAICS `legendFormat` is only missing in one call to `buildLogVolumeQuery`. With other words: the "default" is only used once. Why not move `legendFormat: '{{level}}',` to [that place](https://github.com/grafana/explore-logs/pull/298/files#diff-b45e3639aa418f6d7b7d271784e0e807d5035b6cabee333d3eacee5478cf0fddR217) and remove `legendFormat` from the defaults? 

Same for `buildLogQuery`. The default is only used once: https://github.com/grafana/explore-logs/pull/298/files#diff-bf4a86d7b83030323394834b70dc2c1e8c8118278232286ed656b92dfca0e6fcR92

Reply:
I kind of also liked readability - knowing which query is log and which volume, but I was thinking about it and you are right. 

Also having no defaults for `legendFormat` and `maxLines` is technically more correct, because we are defaulting to Loki data source defaults (e.g. maybe someone is setting max lines in data source to 500, and we should respect it). 

So updated in https://github.com/grafana/explore-logs/pull/298/commits/0214a72c90559ac6b106b3c5476db384235d4a69